VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKProduct.h

```diff
@@ -0,0 +1,76 @@
+@import JSONModel;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface VSDKProduct : JSONModel
+
+/**
+  Product identifier (REQUIRED)
```

Comment:
I know it is a bit redundant, but I would prefer to leave it there so that it's really clear that said property is _REQUIRED_. But that is a really personal preference if you want I can remove it.

Reply:
Let's see what @jcazevedo has to say

Reply:
Removed the _REQUIRED_ in 1d3cc3e. It will rely on the `nonnull` property attribute.